compiler/bindings/python/IREECompilerDialectsModule.cpp

```diff
@@ -21,6 +21,32 @@ static const char *kGpuModuleImportPath =
 namespace py = pybind11;
 using namespace mlir::python::adaptors;
 
+std::vector<MlirOperation>
+ireeCodegenGetExecutableVariantOpsBinding(MlirModule module) {
+  size_t numOps = 0;
+  ireeCodegenGetExecutableVariantOps(module, &numOps, nullptr);
+
+  std::vector<MlirOperation> ops(numOps);
+
+  ireeCodegenGetExecutableVariantOps(module, &numOps, ops.data());
+
+  return ops;
+}
+
+std::vector<py::object> ireeCodegenQueryMMAIntrinsicsBinding(MlirOperation op) {
+  size_t numMMAs = 0;
+  ireeCodegenQueryMMAIntrinsics(op, &numMMAs, nullptr);
+  std::vector<uint32_t> mmaIntrinsics(numMMAs);
+  ireeCodegenQueryMMAIntrinsics(op, &numMMAs, mmaIntrinsics.data());
+
+  py::object mmaIntrinsicEnum =
+      py::module_::import(kGpuModuleImportPath).attr("MMAIntrinsic");
+  std::vector<py::object> mmaList(numMMAs);
+  std::transform(mmaIntrinsics.begin(), mmaIntrinsics.end(), mmaList.begin(),
+                 [&](uint32_t rawValue) { return mmaIntrinsicEnum(rawValue); });
```

Comment:
nit: I'd make this a for loop, I don't think the transform helps the readability much...